### PR TITLE
Prepare release r1.1

### LINKED ISSUE #334: Prepare release PR for r1.1
**Problem description**

The release candidate version of QoD APIs are (over)due for the Fall24 meta-release. We need to prepare the release PR parallel to the remaining PR reviews and bug fixes (see #292).

**Expected action**

Create Release PR which will finally:

- [ ] Create the CHANGELOG.md update for all three APIs in the repository
- [ ] Update the README, mentioning the pre-release
- [ ] Update the API Readiness Checklists for all three API as needed
- [ ] Set the version number and base path within all three YAML files

**Additional context**

Prerequisite to finalize and merge the PR is the completion of the issues and PRs within #292 
A good example for "multi-API" release can be found in https://github.com/camaraproject/DeviceStatus/releases/tag/r1.1 (with [release PR ](https://github.com/camaraproject/DeviceStatus/pull/190))

### PR DESCRIPTION
**Pull request by hdamker**
#### What type of PR is this?

Add one of the following kinds:

* subproject management

#### What this PR does / why we need it:

Prepare the release r1.1:

- [x] Update CHANGELOG.med
- [x] Update README.md
- [x] Set versions in quality-on-demand and qos-profiles
- [x] Set version in qos-provisioning (check all "wip", including event version)
- [x] Update API Readiness Checklists

Update - additional fixes

- Aligned sink description between quality-on-demand and qod-provisioning to address #338 

#### Which issue(s) this PR fixes:

Fixes #334 #338

### CODE REVIEW

**Comment by hdamker** [comment]
@RandyLevensalor @jlurien have a look on the r1.1 section in CHANGELOG.md.

File: documentation/API_documentation/qod-provisioning-API-Readiness-Checklist.md
Comment (jlurien) [inline documentation/API_documentation/qod-provisioning-API-Readiness-Checklist.md:12]: Yo can remove the PR: from comments in line 7, as PR should be merged before this one.

Comment (hdamker) [reply documentation/API_documentation/qod-provisioning-API-Readiness-Checklist.md:12]: Done in https://github.com/camaraproject/QualityOnDemand/pull/336/commits/2644cecbb968549d7ffae3531a8386aa3653c307

**Comment by hdamker** [comment]
Release PR is almost complete ... just waiting for the merge of #299 and the ability to set the version in the new API qos-provisioning to 0.1.0-rc.1

File: README.md
Comment (jlurien) [inline README.md:59]:
```suggestion
  * [Registration / Join](https://zoom-lfx.platform.linuxfoundation.org/meeting/94112812156?password=f238d6af-c959-48d7-a862-abdb3c648e40)
```

**Comment by hdamker** [comment]
Replaced all "wip" in qod-provisioning.yaml:
* in version and server field according to guidelines
* in org.camaraproject.qod-provisioning.vwip.status-changed to org.camaraproject.qod-provisioning.v0.status-changed.
  * There is no specific guideline for initial API versions
  * The event version should be decoupled from the API version (as the API can have breaking changes, but keep the event structure stable) ... so if the event structure will change with a later initial API version we can still then use another number.
  * BTW: I would not reset to wip as long as there is no breaking change to the event structure within a PR
* in https://api.example.com/qod-provisioning/v0.1/device-qos/... set to the current one, but as it is only an example, wouldn't matter too much if we miss this one going forward

**Comment by hdamker** [comment]
@camaraproject/quality-on-demand_maintainers @camaraproject/release-management_maintainers 
The release PR for r1.1 is now complete and ready for review. 

Note: #325 has to be merged first before this PR can be merged and the release created.

File: CHANGELOG.md
Comment (RandyLevensalor) [inline CHANGELOG.md:83]: We need to add in the breaking change from a get to a post for querying QoS profiles and the ability to query by device.  Would that go under added or changed?

Since it's a breaking change, it would be good to highlight it as such.

Would something like this work?
```
* Added query to qos-profiles to query profiles available on a given device by @RandyLevensalor in https://github.com/camaraproject/QualityOnDemand/pull/318.
NOTE: This update also changes the `/qos-profiles` from a get to a post to support this new capability.
```

**Review by jlurien — APPROVED** [review]
LGTM

File: README.md
Comment (eric-murray) [inline README.md:30]:
```suggestion
* The latest public release is available here: https://github.com/camaraproject/QualityOnDemand/releases/latest
* For changes see [CHANGELOG.md](https://github.com/camaraproject/QualityOnDemand/blob/main/CHANGELOG.md)
```

**Review by eric-murray — DISMISSED** [review]

**Review by eric-murray — APPROVED** [review]

**Review by rartych — APPROVED** [review]
LGTM

**Review by RandyLevensalor — APPROVED** [review]
LGTM

**Pull request merged**
